vars.md

```diff
@@ -42,4 +42,10 @@ assured that they are persisted.
 | NUM_NODES                | Set the number of virtual machines to be provisioned. This VMs will be further configured as control-plane or worker Nodes      |   | 2 |
 | VM_EXTRADISKS            | Add extra disks to the virtual machines provisioned. By default the size of the extra disk is set in the libvirt Ansible role to 8 GB        | "true", "false" | "false" |
 | DEFAULT_HOSTS_MEMORY     | Set the default memory size in MB for the virtual machines provisioned.        |  | 4096 |
-| CLUSTER_NAME             | Set the name of the target cluster |  | test1 |
\ No newline at end of file
+| CLUSTER_NAME             | Set the name of the target cluster |  | test1 |
+| IRONIC_TLS_SETUP | Enable TLS for Ironic and inspector | "true", "false" | "true" |
+| IRONIC_BASIC_AUTH | Enable HTTP basic authentication for Ironic and inspector | "true", "false" | "true" |
+| IRONIC_CA_CERT_B64 | Base 64 encoded CA certificate of Ironic |  |   |
+| IRONIC_CACERT_FILE | path to the CA certificate of Ironic |  | /opt/metal3-dev-env/certs/ironicCA.pem |
+| IRONIC_CERT_FILE | path to the certificate of Ironic |  | /opt/metal3-dev-env/certs/ironic.crt |
+| IRONIC_KEY_FILE | path to the certificate key of Ironic |  | /opt/metal3-dev-env/certs/ironicCA.key |
```

Comment:
username and password vars for ironic and inspector?

Reply:
ok, will add.